### PR TITLE
Create release notes for PGD v4.3.3

### PR DESCRIPTION
**Pull request by Jagdish-Kewat-EDB**
* Create release notes for PGD v4.3.3 and add entries for CLI and Utilities

### CODE REVIEW

File: product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+EDB Postgres Distributed version 4.3.3 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+!!! Note
+This version is required for EDB Postgres Advanced Server versions 12.14.18, 13.10.14, 14.7.0 and later.
+!!!
+
+| Component | Version |  Type        |                                                   Description                                                           |
+| --------- | ------- | ------------ | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 5.3.0   | Bug&nbsp;fix | Ensure that the WalSender process does not request locks on the partitions, thus avoiding a deadlock with user process waiting on sync commit. (RT97952) |
```
Comment (vadim4o) [inline product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx:16]: 5.3.0? Not 4.3.3?

File: product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| BDR       | 5.3.0   | Bug&nbsp;fix | Only consider CAMO transactions to be asynchronous when the CAMO setup was degraded to local mode. This solves the split brain problem when deciding fate of transactions that happened during failover. (RT78928) |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Handle partitions with different attribute numbers when batch inserting. (RT99115) |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Fix unsafe CAMO decisions in remote_write mode. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Ensure that the taskmgr process does not ignore SIGINT. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Clean up stale records in bdr.taskmgr_local_work_queue during physical join. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Fix a bug in copying bdr.autopartition_rules during logical join. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Override bdr.ddl_replication=off in taskmgr worker. |
+| BDR       | 5.3.0   | Bug&nbsp;fix | Avoid a potential deadlock between `bdr.autopartition_wait_for_partitions()` and taskmgr|
+| BDR       | 5.3.0   | Bug&nbsp;fix | Block new EPAS automatic partitioning on PGD cluster. |
```
Comment (vadim4o) [inline product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx:25]: A line is missing after this line 25 -

Allow existing automatically partitioned tables when cluster is created or upgraded.

- please check 50-Block-EPAS-autopartitioning-with-BDR file in REL_4_STABLE

File: product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx:16]:
```suggestion
| BDR       | 4.3.3   | Bug&nbsp;fix | Ensure that the WalSender process does not request locks on the partitions, thus avoiding a deadlock with user process waiting on sync commit. (RT97952) |
```

**Comment by djw-m** [comment]
Surely all the BDR versions here should be BDR 4.3.3?

**Comment by vadim4o** [comment]
@djw-m I think so, e.g. in https://www.enterprisedb.com/docs/pgd/4/rel_notes/pgd_4.3.2_rel_notes/ all are 4.3.2

File: product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx (hunk 2 above)
Comment (djw-m) [reply product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx]: Fixed

**Comment by djw-m** [comment]
All version numbers updated, missing entry included.

File: product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/4/rel_notes/pgd_4.3.3_rel_notes.mdx]: Fixed

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**